feat(curriculum): add steps for employee profile generator workshop (#64896)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/blocks/workshop-employee-profile-generator/694bf6eb530e19a9c48c59f3.md

---
id: 694bf6eb530e19a9c48c59f3
title: Step 12
challengeType: 20
dashedName: step-12
---

# --description--

Let's practice `str()` conversion one more time. Just like with age, you must convert any numeric variable to a string before concatenating it with other text.

Create a variable `experience_info`. Assign it a string formed by concatenating `'Experience: '`, the integer `5` (converted to a string), and `' years'`. Print the result to the terminal.

# --hints--

> You should create a variable `experience_info` with the value `'Experience: 5 years'`.

Reviewer: ````suggestion

You should have a variable named `experience_info`.

```js
({ test: () => assert(runPython(`_Node(_code).has_variable("experience_info")`)) })
```

You should assign a string formed by concatenating `'Experience: '`, `str(5)`, and `' years'` to your `experience_info` variable.
````

---
id: 694bf6eb530e19a9c48c59f3
title: Step 12
challengeType: 20
dashedName: step-12
---

# --description--

Let's practice `str()` conversion one more time. Just like with age, you must convert any numeric variable to a string before concatenating it with other text.

> Create a variable `experience_info`. Assign it the value of `'Experience: '` concatenated with `5` (converted to a string) and `' years'`. Print the result.

Reviewer: ```suggestion
Create a variable `experience_info`. Assign it a string formed by concatenating `'Experience: '`, the integer `5` (converted to a string), and `' years'`. Print the result to the terminal.
```

# --hints--

> You should create a variable `experience_info` with the value `'Experience: 5 years'`.

Reviewer: ```suggestion
You should create a variable `experience_info` with the value `'Experience: ' + str(5) + ' years'`.
```

---
id: 694bf6eb530e19a9c48c59f3
title: Step 12
challengeType: 20
dashedName: step-12
---

# --description--

Now you're going to use the `str()` function one more time. Just like with age, you must convert any numeric variable to a string before concatenating it with other text.

> Create a variable `experience_info`. Assign it a string formed by concatenating `'Experience: '`, the integer `5` (converted to a string), and `' years'`. Print the result to the terminal.

Reviewer 2: why do I need to do that? wouldn't it be much easier to `experience_info = 'Experience: '5 years'`? If we want to give more practice with concatenating a number inside a string, then we need to do it as a variable again
